### PR TITLE
Fix failing integration tests

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Some integration tests started to fail once `SSL` was enabled (as a security plugin feature).
The proposed changes fixes these test. 2 tests were disabled due to the plugin bugs: https://github.com/opensearch-project/security/issues/1731 and https://github.com/opensearch-project/security/issues/1732. One failing test was deleted as a duplicate of another one.

Should be merged after #19 to avoid conflicts.

Tested with `OpenSearch` **and** `OpenDistro` both **with and without** `SSL` (using this fix https://github.com/Bit-Quill/opensearch-net-abstractions/pull/6), 100% tests pass.

### CODE REVIEW

**Review by MaxKsyunz — APPROVED** [review]

**Review by alexmeizer — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by alexmeizer — APPROVED** [review]

File: tests/Tests/ClientConcepts/Troubleshooting/DeprecationLogging.doc.cs [hunk 1]
```diff
@@ -49,11 +44,13 @@ namespace Tests.ClientConcepts.Troubleshooting
 	 *
 	 * OpenSearch.NET and OSC report these back to you so you can log and watch out for them.
 	 */
+	[SkipVersion(">1.0.0,<1.13.0", "Disabled due to https://github.com/opensearch-project/security/issues/1731.")]
```
Comment (ghost) [inline tests/Tests/ClientConcepts/Troubleshooting/DeprecationLogging.doc.cs]: Should include 1.0.0

File: tests/Tests/Indices/AliasManagement/DeleteAlias/AliasDeleteApiTests.cs [hunk 2]
```diff
@@ -27,13 +27,15 @@
...
 using OpenSearch.Net;
+using OpenSearch.OpenSearch.Xunit.XunitPlumbing;
 using Osc;
 using Tests.Core.ManagedOpenSearch.Clusters;
 using Tests.Framework.EndpointTests;
 using Tests.Framework.EndpointTests.TestState;
 
 namespace Tests.Indices.AliasManagement.DeleteAlias
 {
+	[SkipVersion(">1.0.0", "Disabled due to https://github.com/opensearch-project/security/issues/1732. Affects OpenDistro as well.")]
```
Comment (ghost) [inline tests/Tests/Indices/AliasManagement/DeleteAlias/AliasDeleteApiTests.cs]: Should include 1.0.0

Comment (Yury-Fridlyand) [reply tests/Tests/Indices/AliasManagement/DeleteAlias/AliasDeleteApiTests.cs]: Thank you, updated in 3be308037.

File: tests/Tests/ClientConcepts/Troubleshooting/DeprecationLogging.doc.cs (hunk 1 above)
Comment (Yury-Fridlyand) [reply tests/Tests/ClientConcepts/Troubleshooting/DeprecationLogging.doc.cs]: Thank you, updated in 3be308037.

**Comment by ghost** [comment]
LGTM! Thanks

**Pull request merged**